src/Engine/Component/Shape.cpp

```diff
@@ -8,29 +9,39 @@ namespace Component {
 
         Json::Value concreteShape;
         switch (shape->GetKind()) {
-            case ::Physics::Shape::Kind::Sphere: {
+            case Physics::Shape::Kind::Sphere: {
```

Comment:
Why the brackets?

Reply:
Scoping. I declare local variables that would otherwise leak to subsequent cases.